liquid/conf/views.py

```diff
@@ -6,14 +6,14 @@
 from django.forms import ValidationError
 from conf.models import Company
 from utils.django_mailman.models import List
-from utils.group_decorator import group_admin_required
+from utils.group_decorator import group_admin_required, check_group_admin
 import simplejson as json
 
 def main(request):
 	return render_to_response('conf/main.html',{"section":"conf", "page":"main"},context_instance=RequestContext(request))
 
 def jobfair(request):
-    if request.user.is_authenticated():
+    if request.user.is_authenticated() and check_group_admin(['Conference', '!Company'], request):
```

Comment:
While it'd be nice to get in a fix relatively soon, it's not a do or die kind of thing. Trying to access the page logged in gave me permission denied with no relevant info, while accessing the page while not logged in gave me information about resume book and the login pages. I don't want students looking to upload their resumes to think that the page is broken if they're logged in.

That said, trying to repro this just now is now giving me what I'd imagine is the correct behavior. This hasn't been pushed out yet, has it?